### PR TITLE
Parse version from CHANGELOG.md

### PR DESCRIPTION
**Pull request by DoomHammer**

### CODE REVIEW

File: bennedetto/settings/common.py [hunk 1]
```diff
@@ -84,10 +84,22 @@
...
+
+        reUnreleased = re.compile('^## Unreleased$')
+        reVersion = re.compile('^## ([0-9].*) -')
+
+        for line in f:
+            if reUnreleased.match(line):
+                suffix = '-Unreleased'
+            m = reVersion.match(line)
+            if m:
+                version = m.group(1)
```
Comment (arecker) [inline bennedetto/settings/common.py:101]: Do you think you could modify this to always prepend a `v`, _e.g._ `v0.1`

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Change Log
```
Comment (arecker) [inline CHANGELOG.md:1]: Nice work on the changelog.

File: bennedetto/settings/common.py (hunk 1 above)
Comment (DoomHammer) [inline bennedetto/settings/common.py:101]: [One `v` coming your way, sir!](https://youtu.be/Otv5ywOa-8U?t=10s)

Comment (arecker) [inline bennedetto/settings/common.py:101]: Ha-ha. Nice.

**Pull request merged**